CMakeLists.txt

```diff
@@ -661,11 +669,16 @@ if (PARQUET_BUILD_SHARED)
 endif()
 
 if (PARQUET_BUILD_STATIC)
+    if (MSVC)
+      set(LIB_NAME_STATIC parquet_static)
+    else()
+      set(LIB_NAME_STATIC parquet)
+    endif()
     add_library(parquet_static STATIC $<TARGET_OBJECTS:parquet_objlib>)
     set_target_properties(parquet_static
       PROPERTIES
       LIBRARY_OUTPUT_DIRECTORY "${BUILD_OUTPUT_ROOT_DIRECTORY}"
-      OUTPUT_NAME "parquet")
+      OUTPUT_NAME ${LIB_NAME_STATIC})
     target_link_libraries(parquet_static
       LINK_PUBLIC ${LIBPARQUET_INTERFACE_LINK_LIBS})
 endif()
```

Comment:
can you adapt the Arrow function for doing this, maybe a little more reusable / readable? https://github.com/apache/arrow/blob/master/cpp/cmake_modules/BuildUtils.cmake#L85